### ISSUE TITLE
Feature request: moving events between calendars

### ISSUE THREAD

**Issue by lonkaars** [etesync/etesync-web#141]
See [moving events between calendars](https://github.com/etesync/server/issues/49)

**Comment by piu-logistics** [comment]
Yes please, just bumping into this issue.  Please allow events to be moved between calenders.
